src/index.js

```diff
@@ -48,11 +48,13 @@ class Conjunction extends Conditions {
   }
 
   toString() {
-    return this.args.length
-      ? this.args.map(arg => "(" + arg + ")").join(" and ")
-      : "";
-  }
+    if (!this.args.length) return "";
 
+    return this.args.map((arg) => {
+      const s = (arg.toString) ? arg.toString() : arg;
+      return "(" + s + ")";
+    }).join(" and ");
```

Comment:
```
const s = (arg.toString) ? arg.toString() : arg;
      return "(" + s + ")";
```
and ` "(" + s + ")" ` is actually the same for javascript (with some exceptions)
![image](https://user-images.githubusercontent.com/4107999/81758086-893c7700-94c9-11ea-96b9-24eca2745070.png)